app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  def retrieve_recent_posts
+    posts.order(:created_at).limit(3)
+  end
+end
```

Comment:
- [ ] kindly add all the relations to your user model i.e `has_many :posts, foreign_key: 'author_id'`